app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -200,13 +196,12 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                         //SharePreference to store image
                         shre = PreferenceManager.getDefaultSharedPreferences(this);
                         SharedPreferences.Editor edit = shre.edit();
-                        edit.putString("image_data", encodedImage);
+                        edit.putString(Constant.IMAGE_DATA, encodedImage);
```

Comment:
Ideally, only those string resources should go into Strings.xml which are visible to the user.
Here, "image_data" is used as a key and not visible to the user, so there is no need of moving it to Strings.xml